### ISSUE TITLE
Multiple calls to setMetadata should not append to private metadata list

### ISSUE THREAD

**Issue by senocular** [pabloalmunia/javacriptdecorators#6]
As per https://github.com/tc39/proposal-decorators/issues/424#issuecomment-894672853 multiple calls to setMetadata through multiple decorators applied to the same private field should not result in multiple instances of the metadata in the resulting private array.  Example:

```js
const DECO = Symbol();
 
function deco(value, { setMetadata, name }) {
  setMetadata(DECO, name);
}

class A {
  @deco
  @deco
  #field;
}

console.log(A.prototype[Symbol.metadata][DECO].private)
// Actual: [ #field, #field]
// Expected: [ #field ]
```

**Comment by pabloalmunia** [comment]
Thanks Trevor for the help and apologies for our mistakes.

We have just released version 0.5.0 on the web that fixes this bug.

**Comment by senocular** [comment]
Thanks! confirmed fixed on 0.5.0
